protocol/x/perpetuals/keeper/perpetual.go

```diff
@@ -870,8 +870,8 @@ func (k Keeper) GetMarginRequirements(
 	return bigInitialMarginQuoteQuantums, bigMaintenanceMarginQuoteQuantums, nil
 }
 
-// GetSettlement returns the net settlement amount (in quote quantums) given the
-// perpetual Id and position size (in base quantums).
+// GetSettlement returns the net settlement amount ppm (in quote quantums) given
```

Comment:
Accordingly, can you rename method to `GetSettlementPpm` to avoid downstream confusion